### PR TITLE
Qradar/playbooks

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
Ready

## Description
The new playbooks are used to replace previous scripts by the same name, in order to support custom timeout functionality.
## Does it break backward compatibility?
   - No

## Additional changes
Previous scripts were deprecated.

### CODE REVIEW

File: Playbooks/playbook-QRadar_-_Get_offense_correlations_V3_5_0.yml [hunk 1]
```diff
@@ -0,0 +1,328 @@
+id: 'QRadar - Get offense correlations temp'
```
Comment (JonathanMeler) [inline Playbooks/playbook-QRadar_-_Get_offense_correlations_V3_5_0.yml]: temp?

File: Playbooks/playbook-QRadarFullSearch.yml [hunk 2]
```diff
@@ -149,6 +149,8 @@ tasks:
     scriptarguments:
       headers:
         simple: ${inputs.headers}
+      output_path:
+        simple: ${inputs.output_path}
```
Comment (JonathanMeler) [inline Playbooks/playbook-QRadarFullSearch.yml]: Don't use DT use visual filters. 
Relevant for all the playbooks

File: Playbooks/playbook-QRadar_-_Get_offense_correlations_.yml [hunk 3]
```diff
@@ -136,118 +197,148 @@ tasks:
...
     task:
-      id: 324f9756-1ddb-4d58-8d45-26f892ba6759
+      id: a2ab59d6-2601-488b-80ab-e67cf8d17196
       version: -1
-      name: Get correlations' logs
-      description: ""
-      scriptName: QRadarGetCorrelationLogs
+      name: Clean up context
+      description: Cleans up the context
+      scriptName: DeleteContext
```
Comment (JonathanMeler) [inline Playbooks/playbook-QRadar_-_Get_offense_correlations_.yml]: What is the DeleteContext for?

**Comment by DeanArbel** [comment]
image

**Comment by DeanArbel** [comment]
image

**Pull request closed without merging**